packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -394,6 +423,75 @@ contract Reserve is
     return _transferGold(to, value);
   }
 
+  /**
+   * @notice Transfer ERC20 token to a whitelisted address subject to reserve spending limits is the limit is set, othersise the limit is 100%.
+   * @param to The address that will receive the ERC20 token.
+   * @param value The amount of ERC20 token to transfer.
+   * @param erc20TokenAddress The token address you're transferring.
+   * @return Returns true if the transaction succeeds.
+   */
+  function transferErc20Token(address erc20TokenAddress, address payable to, uint256 value)
```

Comment:
Similar to comments above: Let's not give CELO a special role and have one function for all (instead of a separate transferGold)